automerge/src/op_set.rs

```diff
@@ -45,13 +45,11 @@ impl OpSetInternal {
     }
 
     pub(crate) fn iter(&self) -> Iter<'_> {
-        let mut objs: Vec<_> = self.trees.keys().collect();
-        objs.sort_by(|a, b| self.m.lamport_cmp(a.0, b.0));
+        let mut objs: Vec<_> = self.trees.iter().collect();
+        objs.sort_by(|a, b| self.m.lamport_cmp((a.0).0, (b.0).0));
```

Comment:
I wonder if it would be worth trying out a btreemap instead of a hashmap for the trees at some point so that we don't have to do an intermediate collect and sort.

Reply:
O like it, although o suppose for that to work we would have to use the `ExId` as a key?

Reply:
Maybe, so it might not be so feasible.